### PR TITLE
Hillary is so cool

### PR DESCRIPTION
**Pull request by DarthHater**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -16,5 +16,6 @@ func main() {
 }
 
 func doAnotherThing(float float32) string {
+	fmt.Println("Hillary is so cool")
```
Comment (designergod420) [inline main.go]: Can you change this to say "Hillary is so friggin cool"

**Review by designergod420 — CHANGES_REQUESTED** [review]
Can you make these changes plz

File: main.go (hunk 1 above)
Comment (DarthHater) [reply main.go]: Ok, I'll do that

Comment (DarthHater) [reply main.go]: In: 2e502db

**Comment by DarthHater** [comment]
@designergod420 mind taking a look again? Made your suggested changes!

**Review by designergod420 — APPROVED** [review]

**Pull request merged**